resources/public/api/conf/2.0/retrieve_income_tax_crystallisation.yaml

```diff
@@ -63,11 +48,21 @@ get:
         </tr>
       </tbody>
     </table>
```

Comment:
Should the description of Dynamic gov-test-scenario be added here or will the sandbox team add that in?

Reply:
yh, the sandbox team will be working on this next